src/devices/sunricher.ts

```diff
@@ -55,6 +67,32 @@ async function syncTime(endpoint: Zh.Endpoint) {
 }
 
 const definitions: Definition[] = [
+    {
+        zigbeeModel: ['HK-SENSOR-4IN1-A'],
+        model: 'HK-SENSOR-4IN1-A',
+        vendor: 'Sunricher',
+        description: '4IN1 Sensor',
+        fromZigbee: [fz.ias_occupancy_alarm_1, fz.battery, fz.temperature, fz.humidity, fz.illuminance],
+        toZigbee: [],
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3, '4': 4, '5': 5}}),
```

Comment:
Try without this, ` meta: {'multiEndpoint': true},` and remove all `endpointNames, e.g. `{'endpointNames': ['3']}),
            humidity({'endpointNames': ['4']}),`